BitmarkRegistry/Helpers/ErrorReporting.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  ErrorReporting.swift
+//  BitmarkRegistry
+//
+//  Created by Anh Nguyen on 7/2/19.
+//  Copyright © 2019 thuyentruong. All rights reserved.
+//
+
+import Foundation
+import Sentry
+
+// Send error report to Sentry
+struct ErrorReporting {
+  
+  // Set current bitmark account to sentry error report to be informative to debug
+  public static func setAccount(bitmarkAccount: String?) {
```

Comment:
it's just my thinking but I think 
- the argument name should be bitmarkAccountNumber / accountAddress; cause sometimes when I read code I kindly lost between bitmarkAccount as Account vs String :) 

- the function can be setUser for matching with implementation?

Reply:
Ok.